simtools/simtel/simulator_array.py

```diff
@@ -111,3 +114,30 @@ def _check_run_result(self, run_number=None):
             raise InvalidOutputFileError(msg)
         self._logger.debug(f"simtel_array output file {output_file} exists.")
         return True
+
+    @staticmethod
```

Comment:
Can you explain the choice of having those values hardwired? Any easy way to make them configurable?

Reply:
I hardcoded them in the same way they are hardcoded in sim_telarray. Since we do not use the sim_telarray histograms, I didn't see a reason to make it configurable. However, if you think it's worth it, please make this comment into an issue and I will implement it after this PR (and use this function to set default values).

Reply:
I think it is easy to miss that this is for histogramming only. Could we either change the function call or at least say in the first line of the docstring `..used for histogramming in sim_telarray`? Quick readers might otherwise thing we hardwire the spectral index for the simulations.

Reply:
Changed the name of the function to get_power_law_for_sim_telarray_histograms.